tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -601,6 +601,12 @@ public static partial class Tags
         /// </summary>
         internal const string AppSecWafInitRuleErrors = "_dd.appsec.event_rules.errors";
 
+        /// <summary>
+        /// A boolean allowing the propagation to downstream services the information that the current distributed trace
+        /// is containing at least one ASM security event, no matter its type (threats, business logic events, IAST, etc.).
+        /// </summary>
+        internal const string PropagatedAppSec = "_dd.p.appsec";
```

Comment:
Consider moving this field into the nested `Propagated` class at the bottom of this file.

Reply:
Changed in 08ca43d